src/pages/Product/Product.js

```diff
@@ -10,19 +14,19 @@ const Product = () => {
       <div className="productDetail">
         <div className="productDetailImg">
           <img
-            src={`/images/golf-ball-.jpg`} // color 이름에 따라 다른 이미지 경로 넣기
+            src={`./../public/images/golf-ball-.jpg`} // color 이름에 따라 다른 이미지 경로 넣기
```

Comment:
```suggestion
            src={`/images/golf-ball-.jpg`} // color 이름에 따라 다른 이미지 경로 넣기
```
해당 이미지의 경로가 public에 있는 이미지기 때문에 위와 같이 작성해도 이미지를 불러올 수 있습니다!